app/routes/wallet/dids.py

```diff
@@ -60,12 +85,20 @@ async def list_dids(
     return did_result.results
 
 
-@router.get("/public", response_model=DID)
+@router.get("/public", response_model=DID, summary="Fetch Public DID")
 async def get_public_did(
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> DID:
     """
-    Fetch the current public DID.
+    Fetch the current public DID
+    ---
+
+    This endpoint allows you to fetch the current public DID.
```

Comment:
If the description is identical to the heading, then we don't need the description.

We can add some tidbit note: by default, only issuers have public DIDs.